lib/puma/cluster.rb

```diff
@@ -238,7 +249,7 @@ def restart
     def phased_restart(refork = false)
       return false if @options[:preload_app] && !refork
 
-      @phased_restart = refork ? :refork : true
+      @pending_phased_restart = refork ? :refork : :restart
```

Comment:
This was kind of a tricky overload before; since doing things like `!= true` are really unclear, just replace a full phased restart with a different but still truthy value.